GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -980,7 +980,7 @@ public void PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> get
                     string pathFilter = BuildPathFilter(_filterInfo.PathFilter);
                     ArgumentBuilder args = RevisionReader.BuildArguments(_filterInfo.CommitsLimit,
                         _filterInfo.RefFilterOptions,
-                        _filterInfo.BranchFilter,
+                        _filterInfo.IsShowFilteredBranchesChecked ? _filterInfo.BranchFilter : string.Empty,
```

Comment:
Why does the property `ByBranchFilter` not suffice? It makes the property `BranchFilter` return the empty string.

Reply:
ByBranchFilter  is set also when IsShowCurrentBranchChecked  is set, but the contents in BranchFilter must then be ignored
I plan to change this in a followup.
First planned to do this in this PR, but I want to fix/disable tests first